Makefile

```
@@ -31,4 +34,13 @@ feed-update-data:
 .PHONY: feed-create-tables
 feed-drop-tables:
 	docker-compose run --no-deps --rm feed python manage.py drop-tables
-.PHONY: feed-drop-tables
\ No newline at end of file
+.PHONY: feed-drop-tables
+r-shell:
+	docker exec -it r-leggo-twitter bash
+.PHONY: r-shell
+r-export-data:
+	docker exec -it r-leggo-twitter bash -c "Rscript /leggo-twitter-dados/code/export_data.R"
+.PHONY: r-export-data
+r-export-data-db-format:
+	docker exec -it r-leggo-twitter bash -c "Rscript /leggo-twitter-dados/code/export_data.R"
```

Aqui não deveria ser o script que processa os dados pro formato do banco?

```suggestion
	docker exec -it r-leggo-twitter bash -c "Rscript /leggo-twitter-dados/code/processor/export_data_to_db_format.R"
```